src/beacon.js

```diff
@@ -100,6 +101,11 @@ export default function Emitter(endpoint, context) {
     window.addEventListener('beforeunload', transmit);
   }
 
+  this.allowFlush = function() {
```

Comment:
Choice of name. If my suggestion is wrong, then the purpose of the method isn't intuitive to me.
```suggestion
  this.unblockAndFlush = function() {
```